<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SlideShowCfg</class>
 <widget class="QWidget" name="SlideShowCfg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>456</width>
    <height>429</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QCheckBox" name="mCbxZoom">
     <property name="text">
      <string>Resi&amp;ze images</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QCheckBox" name="mCbxRandom">
     <property name="text">
      <string>&amp;Random order</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QCheckBox" name="mCbxShowName">
     <property name="text">
      <string>Show &amp;names</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="cbSpacer1">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>10</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QCheckBox" name="mCbxShowPath">
       <property name="text">
        <string>Show &amp;full path</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QCheckBox" name="mCbxRandomPosition">
     <property name="text">
      <string>Random &amp;position</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="textLabel1_2">
       <property name="text">
        <string>&amp;Delay:</string>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
       <property name="buddy">
        <cstring>mDelay</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KIntSpinBox" name="mDelay">
       <property name="specialValueText">
        <string/>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>240</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="6" column="0">
    <spacer name="spacer11">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>19</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="7" column="0" colspan="4">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLineEdit" name="lineEdit_FeedToAdd"/>
       </item>
       <item>
        <widget class="QListWidget" name="listWidget_Feeds">
         <property name="editTriggers">
          <set>QAbstractItemView::AllEditTriggers</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="pushButton_AddFeed">
         <property name="text">
          <string>Add</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_RemoveFeed">
         <property name="text">
          <string>Remove</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="0" column="2" rowspan="7" colspan="2">
    <widget class="QFrame" name="mPreview">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <zorder>tableWidget</zorder>
    </widget>
   </item>
  </layout>
  <zorder>mCbxZoom</zorder>
  <zorder>mCbxRandom</zorder>
  <zorder>mCbxShowName</zorder>
  <zorder>mCbxRandomPosition</zorder>
  <zorder>horizontalLayoutWidget</zorder>
  <zorder>mPreview</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KIntSpinBox</class>
   <extends>QSpinBox</extends>
   <header>knuminput.h</header>
  </customwidget>
 </customwidgets>
 <includes>
  <include location="local">klineedit.h</include>
  <include location="local">kpushbutton.h</include>
 </includes>
 <resources/>
 <connections>
  <connection>
   <sender>mCbxZoom</sender>
   <signal>toggled(bool)</signal>
   <receiver>mCbxRandomPosition</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mCbxShowName</sender>
   <signal>toggled(bool)</signal>
   <receiver>mCbxShowPath</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>50</x>
     <y>65</y>
    </hint>
    <hint type="destinationlabel">
     <x>75</x>
     <y>91</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
